preprocessing/nextclade/tests/test_config.yaml

```diff
@@ -64,7 +64,7 @@ processing_spec:
       type: date
     function: parse_and_assert_past_date
     inputs:
-      date: required_collection_date
```

Comment:
I changed this to ensure the tests work when the input and output field do not have the same name